src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -302,7 +302,18 @@ Paint* LottieLoader::paint()
 
 bool LottieLoader::override(const char* slot)
 {
-    if (!slot || !comp || comp->slots.count == 0) return false;
+    if (!comp || comp->slots.count == 0) return false;
+
+    //reverting whole slots
```

Comment:
Let's say, if there hasn't been any overriding before, there's no need to reset at all.
```
bool LottieLoader::override(const char* slot) {
   if (!comp || comp->slots.count == 0) return false;
   if (!this->overridden) return true;
```